### PR TITLE
Add v1.3.0 of UserJoinLeaveNotifications

### PR DESCRIPTION
**Pull request by badhaloninja**

### CODE REVIEW

**Review by ljoonal — CHANGES_REQUESTED** [review]
❯ ./review.sh "me.badhaloninja.UserJoinLeaveNotifications" "1.3.0"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     24      0 --:--:-- --:--:-- --:--:--    24
SHA256 sums don't match!
Expected: c5029d87001b85d4cc62f120da94afe7e895ebafed04f3ebc0efd8e388c76d61
Actual:   0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5

File: manifest.json [hunk 1]
```diff
@@ -3688,6 +3688,17 @@
                 }
             },
             "versions": {
+                "1.3.0": {
+                    "changelog": "Added config items for notification sounds",
+                    "releaseUrl": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/tag/v1.3.0",
```
Comment (ljoonal) [inline manifest.json:3693]: Also the actual tag doesn't have a "v" prefix so the releaseUrl is 404

Comment (badhaloninja) [reply manifest.json:3693]: Oops, fixed on the github side bc I want to keep my repos in the same format the link is valid now

**Comment by badhaloninja** [comment]
> ❯ ./review.sh "me.badhaloninja.UserJoinLeaveNotifications" "1.3.0" % Total % Received % Xferd Average Speed Time Time Time Current Dload Upload Total Spent Left Speed 100 9 100 9 0 0 24 0 --:--:-- --:--:-- --:--:-- 24 SHA256 sums don't match! Expected: c5029d87001b85d4cc62f120da94afe7e895ebafed04f3ebc0efd8e388c76d61 Actual: 0019dfc4b32d63c1392aa264aed2253c1e0c2fb09216f8e2cc269bbfb8bb49b5

Also I recalculated the hash again a different way with a fresh download from the release and it's still `c5029d87001b85d4cc62f120da94afe7e895ebafed04f3ebc0efd8e388c76d61`

**Comment by ljoonal** [comment]
> Also I recalculated the hash again a different way with a fresh download from the release and it's still 

...huh, weird. Well, seems to check out now, might've been some weird github temporary 404 or something then.

**Review by ljoonal — APPROVED** [review]

**Pull request merged**